clang_github_tracker/sync_raw.py

```
@@ -0,0 +1,150 @@
+            or latest_issue is not None
+            or latest_pr is not None
+        ):
+            clang_state.save_state(
+                last_commit_date=latest_commit,
+                last_issue_date=latest_issue,
+                last_pr_date=latest_pr,
+                merge=True,
+            )
+
+    except (ConnectionException, RateLimitException) as e:
+        logger.exception("clang_github_tracker sync failed: %s", e)
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)